site/pages/view-all-disruptions.page.tsx

```diff
@@ -81,6 +83,29 @@ export interface Filter {
     searchText?: string;
 }
 
+export const getDisruptionStatus = (disruption: SortedDisruption): Progress => {
+    if (disruption.publishStatus === "DRAFT") {
+        return Progress.draft;
+    }
+
+    if (!disruption.validity) {
+        return Progress.closed;
+    }
+
+    const today = getDate();
+    const disruptionEndDate = getSortedDisruptionFinalEndDate(disruption);
+
+    if (!!disruptionEndDate && dateIsSameOrBeforeSecondDate(disruptionEndDate, today)) {
+        if (disruptionEndDate.isBefore(today)) {
+            return Progress.closed;
+        } else {
+            return Progress.closing;
```

Comment:
Having an issue where if today’s date and time and is within 24 hours of the disruption validity (or disruption’s last validity period) end date and time I am getting open status